.github/workflows/publish-docs.yml

```diff
@@ -0,0 +1,67 @@
+name: Publish Docs
+
+on:
+  # Runs whenever there is a release/tag creation
+  release:
+    types: [ "published" ]
+
+  #  push:
+  #    tags:
+  #      - '[0-9]+.[0-9]+.[0-9]+'
```

Comment:
@casid I don't think it makes a difference if we filter for tags or published releases. Which one do you prefer? I think `release -> published` is more explicit.

Reply:
Yeah, that's a lot clearer!